gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -347,7 +354,11 @@ func (p *glusterBlockProvisioner) createVolume(volSizeInt int, blockVol string)
 			}
 			json.Unmarshal([]byte(out), execBlockRes)
 
-			if execBlockRes.User == "" || execBlockRes.AuthKey == "" {
+			if *execBlockRes == nil {
```

Comment:
@wongma7  Actually I have put mandatory field checks later in the code path and in the return function . However thought of having an extra caution here, as I do dereference of this struct pointer in next lines.